app/main.py

```diff
@@ -1,2 +1,92 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(
+            self,
+            initial_capacity: int = 8,
+            load_factor: float = 0.66) -> None:
```

Comment:
In case there are no arguments on the same line as the opening parenthesis, the closing parenthesis should not have any arguments as well.